blogs_notebooks/titanic_data_science_solutions_blog_1.ipynb

```diff
@@ -0,0 +1,492 @@
+{
```

Comment:
<p>There were no instructions on where to put the data.  we should add some</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai-community/showcase/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>